app/views/dashboard/_wiki.html.erb

```diff
@@ -14,7 +14,10 @@
   </div>
 
   <div class="col-xl-4 col-lg-5 col-md-5 col-5">
-    <a href="/wiki/new" class="btn btn-primary btn-sm btn-outline-secondary btn-block" type="button" style="color:white" ><i class="fa fa-plus-circle" style="color:white" ></i> <%= translation('dashboard._wiki.new_page') %></a>
+    <a href="/wiki/new" class="btn btn-primary btn-sm btn-outline-secondary btn-block" type="button" style="color:white; padding-left:5px;" >
+      <i class="fa fa-plus-circle" style="color:white" ></i> 
```

Comment:
Try to use bootstrap class wherever possible. Otherwise, it looks good to me! 🎉 

```
<a href="/wiki/new" class="btn btn-primary btn-sm text-white" type="button" >
      <i class="fa fa-plus-circle text-white" ></i> 
```

Reply:
Thanks! Will keep that in mind henceforth 😅